src/loader.c

```diff
@@ -127,6 +127,35 @@ static char * cstr_get_pair(char *p,
 	return cstr_skip_ws(p);
 }
 
+static bool strings_equal(const char *str_left, const char *str_right)
```

Comment:
Can you add a comment? Something like this
```suggestion
/* Same as strcmp, but handles NULLs. If both sides are NULL this returns true */
static bool strings_equal(const char *str_left, const char *str_right)
```

Reply:
Done with commit [a111808](https://github.com/pgbouncer/pgbouncer/pull/764/commits/a111808a8910e8a14eb0eb57992d198cb0f75c26)